Dart/src/com/jetbrains/lang/dart/ide/runner/test/DartTestEventsConverter.java

```diff
@@ -151,6 +166,11 @@ private boolean handleTestStart(JsonObject obj) throws ParseException {
     return result;
   }
 
+  private static String encodeLocation(String url, Item item) {
```

Comment:
Yes. I forgot to delete it after inlining it.

On Wed, Dec 16, 2015 at 9:55 AM, Alexander Doroshko <
notifications@github.com> wrote:

> In
> Dart/src/com/jetbrains/lang/dart/ide/runner/test/DartTestEventsConverter.java
> <https://github.com/JetBrains/intellij-plugins/pull/316#discussion_r47808406>
> :
>
> > @@ -151,6 +166,11 @@ private boolean handleTestStart(JsonObject obj) throws ParseException {
> >      return result;
> >    }
> >
> > +  private static String encodeLocation(String url, Item item) {
>
> Method is unused. Can it be removed?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JetBrains/intellij-plugins/pull/316/files#r47808406>.
>